src/config.ts

```diff
@@ -47,14 +47,20 @@ export const ZSCORE = new Map<SYMBOL, number>([
   ['BONK', parseFloat(bonkVars[3]) > 0 ? parseFloat(bonkVars[3]) : 2.58],
 ]); // Corresponds to 80% CI by default
 
-// TODO: Make enum
-export const MODE = new Map<SYMBOL, number>([
-  ['BTC', parseFloat(btcVars[4]) > 0 ? parseFloat(btcVars[4]) : 0],
-  ['ETH', parseFloat(ethVars[4]) > 0 ? parseFloat(ethVars[4]) : 0],
-  ['SOL', parseFloat(solVars[4]) > 0 ? parseFloat(solVars[4]) : 0],
-  ['MNGO', parseFloat(mngoVars[4]) > 0 ? parseFloat(mngoVars[4]) : 0],
-  ['BONK', parseFloat(bonkVars[4]) > 0 ? parseFloat(bonkVars[4]) : 0],
-]); // 0 - Normal 1 - Gamma+Back 2 - Gamma+Back w/ Strike Adjustment 3- Back Only
+export enum ScalperMode {
```

Comment:
If we have to type these in to the cli each time lets make them shorter.

Full
Gamma
Pin
Back

Reply:
enum maps to integers, so there still is parsefloat.

We also should remove the way the cli is setup soon. Should use mostly default arguments